### PR TITLE
Looking forward to pass as a collaborator

### PR DESCRIPTION
**Pull request by GrandFunkyGamer**

### CODE REVIEW

**Comment by muan** [comment]
Hi! Before doing this, please follow the contributor steps first https://github.com/muan/hello-world#not-sure-what-to-do.

**Review by muan — CHANGES_REQUESTED** [review]
See the previous comment.

**Comment by spiiritual** [comment]
looks all good, should be ready for merging by muan

File: MAINTAINERS.md [hunk 1]
```diff
@@ -13,3 +13,5 @@
 - [lee.wp14](https://github.com/leewp14)
 - [genesis331](https://github.com/genesis331)
 - [Ibtehaz](https://github.com/ibtehaz-shawon)
+- [Richard](https://github.com/grandfunkygamer)
```
Comment (muan) [inline MAINTAINERS.md]: This should be your username. I know the last one isn't but that was a mistake too.

**Review by muan — CHANGES_REQUESTED** [review]

File: MAINTAINERS.md (hunk 1 above)
Comment (GrandFunkyGamer) [reply MAINTAINERS.md]: I'm a bit confused here, could you please be specific ?

Comment (SubhrajyotiSen) [reply MAINTAINERS.md]: You need to replace 'Richard' with your username

File: humans/richard.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Hi, I'm Richard Abrokwah from Ghana.
```
Comment (muan) [inline humans/richard.txt]: Apologies that I missed this in the first pass. This file would also need to be named as your username: `grandfunkygamer.txt`.

**Review by muan — CHANGES_REQUESTED** [review]

**Review by muan — APPROVED** [review]
Thanks!

**Pull request merged**